searchlib/src/vespa/searchlib/query/streaming/querynode.cpp

```diff
@@ -263,7 +263,11 @@ QueryNode::build_weighted_set_term(const QueryNodeResultFactory& factory, Simple
 std::unique_ptr<QueryNode>
 QueryNode::build_phrase_term(const QueryNodeResultFactory& factory, SimpleQueryStackDumpIterator& queryRep)
 {
-    auto phrase = std::make_unique<PhraseQueryNode>(factory.create(), queryRep.getIndexName(), queryRep.getArity());
+    vespalib::string index = queryRep.getIndexName();
+    if (index.empty()) {
+        index = DEFAULT;
```

Comment:
I suspect you need to change to SimpleQueryStackDumpIterator::DEFAULT_INDEX due to https://github.com/vespa-engine/vespa/pull/30198 merged just now.

Reply:
This PR has now been rebased to account for the change in #30198